rpi-gpio.js

```diff
@@ -181,7 +181,14 @@ function Gpio() {
                 exportPin(pinForSetup, next);
             },
             function(next) {
-                setEdge(pinForSetup, edge, next);
+              async.retry({times: 10, interval: 100},
+                function(cb){
+                  setEdge(pinForSetup, edge, cb);
```

Comment:
Would this be better put inside the `setEdge()` function wrapping the `fs.write()` call?